sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -685,6 +685,12 @@ def _add_argparse_args(cls, parser):
         help='The time to wait (in milliseconds) for test pipeline to finish. '
              'If it is set to None, it will wait indefinitely until the job '
              'is finished.')
+    # This option is used by Dataflow Runner's Pub/Sub client. Its camelCase
```

Comment:
Would it make sense to use a new TestDataflowOptions class?

Reply:
Done.